.travis.yml

```diff
@@ -27,20 +26,26 @@ install:
 - nohup npm run httpbin &
 - if [ "$CXX" = "g++" ]; then export CXX="g++-4.9" CC="gcc-4.9"; fi
 - gcc -v
-- if ! git clone --depth 1 https://github.com/ni-kismet/emscripten-sdk.git; then cd emscripten-sdk && git remote update -p && git merge --ff-only origin/master && cd ..; fi
-- emscripten-sdk/emsdk activate --build=Release sdk-master-64bit
+- mkdir -p emsdk-tools
+- cp make-it/travis-support/libstdc++6_5.1.0-0ubuntu11~10.04.2_amd64.deb emsdk-tools/
+- cd emsdk-tools && ar xv libstdc++6_5.1.0-0ubuntu11~10.04.2_amd64.deb && cd ..
+- tar -xzf emsdk-tools/data.tar.gz -C emsdk-tools
+- git clone https://github.com/juj/emsdk.git
+- cd emsdk && git checkout b2ebcffbbbc2ba64d5697dc5c37fed530bafecda && cd ..
```

Comment:
What's this commit #?  Does it have a readable named tag you could use instead?

Reply:
Unfortunately the emsdk repo doe not use tags :/

The emsdk tool recommends always using the latest from master and running the `emsdk update` command does that. Unfortunately that has broken our builds before so using a specific commit version.

This is the same commit version used in appveyor.yml so at least now we are using the same version of the emsdk tooling and emscripten libraries across both CIs